db/migrate/20200428143000_create_network_service.rb

```diff
@@ -0,0 +1,29 @@
+# rake db:migrate:down VERSION=20200428000001
+# rake db:migrate:up VERSION=20200428000001
+class CreateNetworkService < ActiveRecord::Migration[5.0]
+  def change
+    create_table :network_services do |t|
+      t.string     :type
+      t.string     :name
+      t.string     :description
+      t.string     :ems_ref
+      t.references :ems,               :type => :bigint
+      t.references :cloud_tenant,      :type => :bigint   
+    end
+
+    create_table :network_service_entries do |t|
+      t.string     :type
+      t.string     :name
+      t.string     :description      
+      t.string     :ems_ref
+      t.references :ems,               :type => :bigint
+      t.references :network_service,   :type => :bigint
+      t.string     :protocol
+      t.string     :source_ports
```

Comment:
What will the format for this be?  If there's multiple ports are the e.g. comma separated?

Reply:
Yes, it will be comma separated

Reply:
Does it makes sense to make it an array column?

Reply:
(Note that comma-separated is fine as well, just brainstorming if maybe array would be useful)

Reply:
It's only for display purposes.